.github/workflows/nanovdb.yml

```diff
@@ -105,10 +105,10 @@ jobs:
           # static build of blosc from vcpkg does not build internal sources.
           # USE_STATIC_DEPENDENCIES is required for IlmBase/OpenEXR defines and
           # Boost as both shared and static libs are installed.
-          - { vc: 'x64-windows-static', build: 'Release', cmake: '-A x64 -G \"Visual Studio 17 2022\" -DOPENVDB_CORE_SHARED=OFF -DUSE_STATIC_DEPENDENCIES=ON -DBLOSC_USE_EXTERNAL_SOURCES=ON' }
-          - { vc: 'x64-windows-static', build: 'Debug',   cmake: '-A x64 -G \"Visual Studio 17 2022\" -DOPENVDB_CORE_SHARED=OFF -DUSE_STATIC_DEPENDENCIES=ON -DBLOSC_USE_EXTERNAL_SOURCES=ON' }
-          - { vc: 'x64-windows',        build: 'Release', cmake: '-A x64 -G \"Visual Studio 17 2022\" -DOPENVDB_CORE_STATIC=OFF' }
-          - { vc: 'x64-windows',        build: 'Debug',   cmake: '-A x64 -G \"Visual Studio 17 2022\" -DOPENVDB_CORE_STATIC=OFF' }
+          - { vc: 'x64-windows-static', crt: 'MT',  build: 'Release', cmake: '-A x64 -G \"Visual Studio 17 2022\" -DOPENVDB_CORE_SHARED=OFF -DUSE_STATIC_DEPENDENCIES=ON -DBLOSC_USE_EXTERNAL_SOURCES=ON' }
+          - { vc: 'x64-windows-static', crt: 'MTd', build: 'Debug',   cmake: '-A x64 -G \"Visual Studio 17 2022\" -DOPENVDB_CORE_SHARED=OFF -DUSE_STATIC_DEPENDENCIES=ON -DBLOSC_USE_EXTERNAL_SOURCES=ON' }
+          - { vc: 'x64-windows',        crt: 'MD',  build: 'Release', cmake: '-A x64 -G \"Visual Studio 17 2022\" -DOPENVDB_CORE_STATIC=OFF' }
+          - { vc: 'x64-windows',        crt: 'MDd', build: 'Debug',   cmake: '-A x64 -G \"Visual Studio 17 2022\" -DOPENVDB_CORE_STATIC=OFF' }
```

Comment:
Adding this 'crt' entry here won't do anything?

Reply:
It's still failing, even if I add 'crt'. I wonder if I can pass `CMAKE_MSVC_RUNTIME_LIBRARY` to the cmake flag per this post: https://discourse.cmake.org/t/msvc-runtime-library-ignored-on-command-line/1644.

Reply:
Can you apply this change ontop of whatever changes you're making where the build breaks so it's clear what this is addressing?

Reply:
It's addressing this issue: https://github.com/apradhana/openvdb/actions/runs/6660668233/job/18102225571

Reply:
Instead of using 'crt', I passed the tag `-DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded` explicitly. It seems to fix the problem: https://github.com/apradhana/openvdb/actions/runs/6662495109

Reply:
Adding a variable in the strategy list (in this case 'crt') won't do anything by itself. It's an entry into the CI matrix which you can then use in the job however you want, its not a special value that the CI is reading in any way. 

I've just taken a look and know whats happening, you need to apply the `VDB_MSVC_RUNTIME_SELECTION` variable to .cu files if you want o fix this natively. Take a look at the `NANOVDB_USE_CUDA` branch in `openvdb/nanovdb/nanovdb/examples/benchmark/CMakeLists.txt`

Reply:
Thanks for the pointer, Nick. For the time being, is it OK if we changed the following two lines in `nanovdb.yml`:
```
          - { vc: 'x64-windows-static', build: 'Release', cmake: '-A x64 -G \"Visual Studio 17 2022\" -DOPENVDB_CORE_SHARED=OFF -DUSE_STATIC_DEPENDENCIES=ON -DBLOSC_USE_EXTERNAL_SOURCES=ON -DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded' }
          - { vc: 'x64-windows-static', build: 'Debug',   cmake: '-A x64 -G \"Visual Studio 17 2022\" -DOPENVDB_CORE_SHARED=OFF -DUSE_STATIC_DEPENDENCIES=ON -DBLOSC_USE_EXTERNAL_SOURCES=ON -DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreadedDebug' }
```
This is what is written in the code right now (after I deleted the 'crt' variable in the strategy list). In place of that, I passed `-DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded` and `DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreadedDebug` for static and static-debug build respectively. Cf this [link](https://github.com/apradhana/openvdb/blob/vdb11_nanovdb_ci/.github/workflows/nanovdb.yml/#L108-#L109).